docker-vertica-v2/packages/cleanup.sh

```diff
@@ -66,8 +66,21 @@ rm -rf \
     /home/dbadmin/.ssh \
     /opt/vertica/sbin/ssh_config \
     /opt/vertica/share/binlib/util/create-or-export-ssh-key \
-    /opt/vertica/share/binlib/util/install-ssh-key 
+    /opt/vertica/share/binlib/util/install-ssh-key
 
+# removing admintools and supported libraries(vbr, agent, scrutinize)
+rm -rf \
+    /opt/vertica/bin/vbr* \
+    /opt/vertica/share/vbr \
+    /opt/vertica/bin/scrutinize \
+    /opt/vertica/agent \
+    /opt/vertica/config/logrotate/agent.logrotate \
+    /opt/vertica/sbin/vertica_agent* \
+    /opt/vertica/config/admintools* \
+    /opt/vertica/bin/admintools \
```

Comment:
A few other things that I think we can remove:
- `/opt/vertica/bin/adminTools`
- `/opt/vertica/sbin/update_vertica`
- `/opt/vertica/sbin/install_vertica`
- `/opt/vertica/config/apikeys.dat`